plane/source/docs/guide-elevon-plane.rst

```diff
@@ -0,0 +1,147 @@
+.. _guide-elevon-plane:
+
+=============
+Elevon Planes
+=============
+
+Elevon planes (also known as delta-wings) are popular for their
+simplicity and robustness.
+
+A typical elevon plane will have 3 servo outputs. One will be the left
+elevon, another for right elevon and the 3rd for throttle.
+
+The recommended setup for APM:Plane with elevon planes is:
+
+.. raw:: html
+
+   <table border="1" class="docutils">
+   <tr><th>Parameter</th><th>Value</th><th>Meaning</th></tr>
+   <tr><td>SERVO1_FUNCTION</td><td>77</td><td>Left elevon</td></tr>
+   <tr><td>SERVO2_FUNCTION</td><td>78</td><td>Right elevon</td></tr>
+   <tr><td>SERVO3_FUNCTION</td><td>70</td><td>Throttle</td></tr>
+   </table>
+
+.. warning:: You should remove the propeller from your aircraft before
+             starting your setup.
+
+Setting Up Your Plane
+=====================
+
+After you have setup your :ref:`RC inputs <rc-throw-trim>`, the next
+step is to setup your 3 outputs.
+
+You can connect your 3 servo cables to any output of your autopilot,
+although using the defaults listed above is recommended.
+
+Servo Reversal
+==============
+
+The next step is to get the reversals right. You should connect the
+battery (with propeller removed) and turn on your RC transmitter. Now
+switch to MANUAL mode and disable the output safety (if enabled).
```

Comment:
I assume because you need to see the surfaces physically move. A reference directly to the button might help it make more sense though.